src/util/motionModel/draftFrameKm.ts

```diff
@@ -28,6 +28,7 @@ export class DraftFrameKm {
   totalDistance = 0;
   highSpeedRecordsInARow = 0;
   lowSpeedRecordsInARow = 0;
+  isStationary = false;
```

Comment:
this entire logic in draftFrameKm is not needed here anymore!

Reply:
kept for later use